translations/TablerBundle.en.xlf

```diff
@@ -110,6 +110,26 @@
         <source>Take me home</source>
         <target>Take me home</target>
       </trans-unit>
+      <trans-unit id="948fezF"  resname="Password Reset Email Sent">
+        <source>Password Reset Email Sent</source>
+        <target>Password Reset Email Sent</target>
+      </trans-unit>
+      <trans-unit id="5FE896d" resname="Enter your email address and your password will be reset and emailed to you">
+        <source>Enter your email address and your password will be reset and emailed to you</source>
+        <target>Enter your email address and your password will be reset and emailed to you</target>
```

Comment:
Is that really how it works? Is the password reset to something automatically generated and emailed to the user? Or does the user get a link to a form, where he can change the password manually?

Reply:
No it's not..
I'm also lost with the default translation but it is what it is.

To answer:
It will complete two field in the user data:
   a. Token
   b. DateTime to when it was ask

'a' is for the link sent from the email
'b' is used to avoid spamming reset form (and email) and link life duration

When clicking on the email link, you will be prompt to change the user password.
It is not because you asked to reset the password, that your password is already reset.
You can ask to reset, but still continue to log in with your old password, until you complete the form to reset.

Reply:
That's what I guessed and already added to the german translation. Lets change the default translation to something more useful...

```suggestion
        <target>Enter your email address and you will be emailed a link that allows to reset your password</target>
```

Reply:
All translations changed